rfc_code_review/comment_summary.md

```diff
@@ -0,0 +1,54 @@
+# RFC: Code Review Comments Summary
+
+## GitHub Missing Features
+
+* Intuitive Review Comments:
+  * While this feedback may be somewhat subjective and depend on personal preferences,
+    it was a common issue that was mentioned by many community members.  For example,
+    some comments disappear when a patch is updated, sometimes comments are auto-hidden
+    and can be easy to miss.  And it can be difficult to distinguish between general
+    review comments and comments on specific lines of code
+* Automatic Subscription Rules (Herald Rules)
+  * There is no builtin way for people to automatically subscribe to a pull request covering
+    a specific part of the codebase.  This is seen as a major downside of GitHub pull requests.
+* Configurable Notifications.
+  * Configuration for notifications is not very flexible and it can be difficult to manage especially
+     for contributors that subscribe to multiple projects.
+* Good support for "Stack Reviews".
+  * "Stack Reviews" are reviews of multiple dependant patches.  This is something that is easy to do
+     with Phabricator, but difficult to do with GitHub.
+
+## GitHub Positive Featues
+
+* Familiar for new contributors.
+  * Phabricator is seen by some as a barrier of entry for new contributors.
+* Can enforce developer policy / restrict pushes.
+  * With GitHub pull requests we can enforce parts of the developer policy with automated checks.
```

Comment:
What's a better way to phrases this ?  s/Can/We could if we wanted to/

Reply:
This can't be in a "GitHub positive Features" IMO: because this isn't part of the A/B comparison of the review tooling **within the bounds of our process**.

You may want to add another specific section like `## Others` at the end to add something like that indeed "If we wanted to change our workflow in the future, Pull-Requests would allow us to enforce that code contributions are going through mandatory automated checks" or something like that.